Composer/packages/lib/shared/__tests__/lgUtils/parsers/parseLgTemplateRef.test.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+
+import { LgTemplateRef } from '../../../src';
+import parseLgTemplateRef from '../../../src/lgUtils/parsers/parseLgTemplateRef';
+
+describe('parseLgTemplateRef', () => {
+  it('should return null when inputs are invalid', () => {
+    expect(parseLgTemplateRef('')).toEqual(null);
```

Comment:
parseLgTempalteRef or extractLgTemplateRefs should be enough. we don't need a extractLgNames